src/main/resources/logback.xml

```diff
@@ -0,0 +1,26 @@
+<configuration>
+    <import class="ch.qos.logback.classic.encoder.PatternLayoutEncoder" />
+    <import class="ch.qos.logback.core.ConsoleAppender" />
+
+    <property name="LOG_PATH" value="target"/>
```

Comment:
It is better to use separate dir in scope of the `target`... You can name this dir as.., `log`, for example...

Reply:
fix